nautobot/apps/api.py

```diff
@@ -9,26 +9,100 @@
     ValidatedModelSerializer,
     WritableNestedSerializer,
 )
+from nautobot.core.api.fields import (
+    ChoiceField,
+    ContentTypeField,
+    NautobotHyperlinkedRelatedField,
+    ObjectTypeField,
+    SerializedPKRelatedField,
+    TimeZoneSerializerField,
+)
+from nautobot.core.api.mixins import WritableSerializerMixin
+from nautobot.core.api.pagination import OptionalLimitOffsetPagination
+from nautobot.core.api.parsers import NautobotCSVParser
+from nautobot.core.api.renderers import FormlessBrowsableAPIRenderer, NautobotCSVRenderer, NautobotJSONRenderer
 from nautobot.core.api.routers import OrderedDefaultRouter
-from nautobot.core.api.views import ModelViewSet, ReadOnlyModelViewSet
-from nautobot.extras.api.mixins import (
-    TaggedModelSerializerMixin,
+from nautobot.core.api.schema import NautobotAutoSchema
+from nautobot.core.api.serializers import (
+    OptInFieldsMixin,
+    TreeModelSerializerMixin,
+)
+from nautobot.core.api.utils import (
+    dict_to_filter_params,
+    dynamic_import,
+    get_api_version_serializer,
+    get_nested_serializer_depth,
+    get_relation_info_for_nested_serializers,
+    get_serializer_for_model,
+    get_view_name,
+    is_api_request,
+    nested_serializer_factory,
+    nested_serializers_for_models,
+    rest_api_server_error,
+    return_nested_serializer_data_based_on_depth,
```

Comment:
Also hesitant about some of the `nested` stuff, do these need to be called directly by apps at any point or are they part of the internal magic that apps don't need to care about?

Reply:
In 2.0 I don't see why the apps would care to mess with nesting serializers. FWIW I don't see any of these being imported in the apps in the `nautobot` org